app/code/Magento/Directory/Model/Currency.php

```diff
@@ -88,6 +95,7 @@ public function __construct(
         \Magento\Framework\Locale\CurrencyInterface $localeCurrency,
         \Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
         \Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null,
+        CurrencySystemConfig $currencyConfig = null,
```

Comment:
Should this not be after the `$data` otherwise it will break backwards compatibility?

Reply:
Fixed.